packages/docs-theme/src/tags/reactDocs.tsx

```diff
@@ -19,15 +19,15 @@ export class ReactDocsTagRenderer {
      * it to an actual component class in the given map, or in the default map which contains
      * valid docs components from this package. Provide a custom map to inject your own components.
      */
-    public render: TagRenderer = ({ value: componentName }, key) => {
+    public render: React.SFC<ITag> = ({ value: componentName }) => {
```

Comment:
why limit this to SFCs? prefer `React.ComponentType<ITag>`

Reply:
this is the type of the `render` function, not the type of the component, so it's actually correct here. but i'll make the change elsewhere to support SFCs in docs components.